CMakeLists.txt

```diff
@@ -3297,8 +3297,15 @@ if(QML)
   list(APPEND QT_EXTRA_COMPONENTS "QuickShapesPrivate")
   list(APPEND QT_EXTRA_COMPONENTS "QuickTemplates2")
   list(APPEND QT_EXTRA_COMPONENTS "QuickWidgets")
-  list(APPEND QT_EXTRA_COMPONENTS "QmlWorkerScript")
   list(APPEND QT_EXTRA_COMPONENTS "ShaderTools")
+  if(QT_VERSION VERSION_GREATER 6.6)
+    list(APPEND QT_EXTRA_COMPONENTS "QuickControls2Basic")
+    list(APPEND QT_EXTRA_COMPONENTS "QuickControls2BasicStyleImpl")
+    list(APPEND QT_EXTRA_COMPONENTS "QuickControls2Fusion")
+    list(APPEND QT_EXTRA_COMPONENTS "QuickControls2FusionStyleImpl")
+  else()
+    list(APPEND QT_EXTRA_COMPONENTS "QmlWorkerScript")
```

Comment:
why else? Did they remove `QmlWorkerScript`? Or is it part of some other package now?

Reply:
QmlWorkerScript is still there as an independent componenet, but is also already installed as a dependency of another component. Without the else branch WIX fails with complains that the QmlWorkerScript.dll is already installed. I have not investigated which component does it.

Reply:
Thanks for the clarification. Maybe add that as a comment or in the commit message?

Reply:
It is probably this commit. 
https://github.com/qt/qtdeclarative/commit/c6057cb4eb4999f341e2ae65d3bf4c76421dfc68